perfkitbenchmarker/providers/aws/aws_dynamodb.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2018 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing class for AWS' dynamodb tables.
+
+Tables can be created and deleted.
+TODO: derive region from endpoint.
+"""
+
+import json
+import logging
+
+from perfkitbenchmarker import resource, errors
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import vm_util
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('aws_dynamodb_region',
+                    'us-west-1',
+                    'The config for the DynamoDB instance.')
+
+
+class AwsDynamoDBInstance(resource.BaseResource):
+
+  def __init__(self, table_name, primary_key, throughput):
+    super(AwsDynamoDBInstance, self).__init__()
+    self.attributes = 'AttributeName=' + primary_key + ',AttributeType=S'
+    self.table_name = table_name
+    self.primary_key = 'AttributeName=' + primary_key + ',KeyType=HASH'
```

Comment:
support for range key?

Reply:
im not sure how many people use range key for performance testing, i havn't been.